### ISSUE TITLE
Disable ipv6 in address list

### ISSUE THREAD

**Issue by ListenNjoy** [crowdsecurity/cs-blocklist-mirror#12]
Is it possible to disable ipv6 from being in the list.?

Best regards
Martin

**Comment by williamdes** [comment]
For sure, but please explain why and what do you use that needs ipv6 removed ?
It's to provide the best features to the users and not a weird workaround

**Comment by williamdes** [comment]
@LaurenceJJones what do you think?

**Comment by williamdes** [comment]
> Going to be honest, we cant filter straight from the decision list so you would have to iterate over the slice and evaluate if ipv6 or ipv4 at request time so response time will be impacted by this. Sort of how you @williamdes did the detection for microtik.

yup, but now is the question whether query string or yaml configuration setting

**Comment by LaurenceJJones** [comment]
> yup, but now is the question whether query string or yaml configuration setting

IMHO I would like this to be a query string. However, when I get time to do V2 I would make it a middle ware in the request chain. So all "formatters" can benefit without having to implement it.

**Comment by williamdes** [comment]
Awesome

**Comment by LaurenceJJones** [comment]
Issue been open for some time. Classing as resolved but options will be available from next release.
